pontoon/base/models.py

```diff
@@ -2911,7 +2911,13 @@ def for_project_locale(
 
     @classmethod
     def map_entities(
-        cls, locale, preferred_source_locale, entities, visible_entities=None
+        cls,
+        locale,
+        preferred_source_locale,
+        entities,
+        visible_entities=None,
+        sibling=False,
+        display_siblings=False,
```

Comment:
The backend will always pass `display_siblings` as `False` to frontend, so to save on web traffic it should instead not pass anything and the frontend would just assume the default value of `False`.

Even better, I believe the logic to show/hide the reveal button would be cleaner if you put something like `areSiblingsRevealed` in a state, similarly to how we handle [`isDiffVisible`](https://github.com/mozilla/pontoon/blob/0f2382606f9a892c670133ae2d4f295e5603c3a4/frontend/src/modules/history/components/Translation.tsx#L39).